slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -30,27 +30,28 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
         std::lock_guard<std::mutex> lk(viam_response_mutex);
         global_pose = latest_global_pose;
     }
+
+    // Rotate pose to XZ plane
+    auto rotated_vector = pcdRotation * global_pose.translation();
```

Comment:
This is a lot easier for me to read than the original version. Thanks for making this change. 👏